models/modules/classifiers.py

```diff
@@ -24,12 +24,20 @@ def __init__(self, input_nc, ndf, nclasses,img_size, norm_layer=nn.BatchNorm2d):
         for n in range(log_size-2):
             nf_mult_prev = nf_mult
             nf_mult = min(2**n, 8)
-            sequence += [
-                nn.Conv2d(ndf * nf_mult_prev, ndf * nf_mult,
-                          kernel_size=kw, stride=2),
-                norm_layer(ndf * nf_mult, affine=True), #beniz: pb with dimensions with batch_size = 1
-                nn.LeakyReLU(0.2, True)
-            ]
+            if n != log_size-3:
```

Comment:
If I understand correctly, we don't add a batchnorm for the last layer?
Maybe add a variable `last_layer_id = log_size - 3` then if `n != last_layer_id` to make it more understandable

Reply:
ok !